aws/main.mk

```diff
@@ -35,7 +35,7 @@ CMD_LOCALSTACK_DOWN ?= @ ( $(DOCKER) rm $$($(DOCKER) stop $$($(DOCKER) ps -a -q
 
 LOCALSTACK_CONTAINER_IP ?= $$($(DOCKER) ps | grep "localstack" > /dev/null && echo "$(LOCALSTACK_IP)" || echo "")
 LOCALSTACK_IP ?= $$($(DOCKER) inspect --format='{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' localstack)
-AWS_ARGS ?= $$(if [ "$(ENABLE_LOCALSTACK)" = "1" ] && [ $(LOCALSTACK_CONTAINER_IP) ]; then echo "--endpoint-url=http://$(LOCALSTACK_CONTAINER_IP):4566"; else echo ""; fi)
+AWS_ARGS ?= $$(if [ "$(ENV)" = "localstack" ] && [ $(LOCALSTACK_CONTAINER_IP) ]; then echo "--endpoint-url=http://$(LOCALSTACK_CONTAINER_IP):4566"; else echo ""; fi)
```

Comment:
`==` vs `=`? Sounds like it shoult be `==` unless I'm mistaken

Reply:
@AutomationD I've decided to use only `$(ENV)` instead of addition `LOCALSTACK_ENV = localstack` variable to not overload ICMK by variables :) . Re: == vs = I tested it - it works as expected (prove https://unix.stackexchange.com/questions/168284/unix-test-when-to-use-eq-vs-vs-in-test-commands :)